velox/expression/tests/utils/FuzzerToolkit.h

```diff
@@ -57,4 +57,46 @@ void sortSignatureTemplates(std::vector<SignatureTemplate>& signatures);
 void compareExceptions(
     std::exception_ptr exceptionPtr,
     std::exception_ptr otherExceptionPr);
+
+// Parse the comma separated list of function names, and use it to filter the
+// input signatures. Return a signature map that (1) only include functions
+// appearing in onlyFunctions if onlyFunctions is non-empty, and (2) not include
+// any functions appearing in skipFunctions if skipFunctions is non-empty.
+template <typename SignatureMapType>
```

Comment:
Would you add @tparam comment to explain what are acceptable types for SignatureMapType ?